addons/common/XEH_preInit.sqf

```diff
@@ -215,6 +215,7 @@ PREP(getConfigGunner);
 PREP(getConfigCommander);
 PREP(getHitPoints);
 PREP(getHitPointsWithSelections);
+PREP(getSelectionsWithoutHitPoints);
```

Comment:
But those are only the selections that don't have any hitpoint assigned, not all damageable selections.